myblog/views.py

```diff
@@ -1,11 +1,12 @@
 from django.shortcuts import render
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 from django.views import View
-from . import models
+from . import models, forms
+from django.urls import reverse
 
 class index(View):
```

Comment:
É mais comum e encorajamos usar nome das classses em camel case, entao ficaria
```suggestion
class Index(View):
```
enquanto as funcoes continnuam em snake case.

Reply:
o mesmo para o detail

Reply:
e NewPost